include/ola/thread/Thread.h

```diff
@@ -33,18 +33,6 @@
 
 #include <string>
 
-#if defined(_WIN32) && defined(__GNUC__)
```

Comment:
I moved it: https://github.com/OpenLightingProject/ola/commit/c30041c1095246ae578c66bc53f5a8f464052e32 
@lukase added it initially: https://github.com/OpenLightingProject/ola/commit/5f4aa5ed067012093b058d30afa6b3f32847fe28

So it needs to stay @sl1200mk2 , by all means add some more ifdefs if its breaking your combination of libraries.